command/git.go

```diff
@@ -310,7 +341,7 @@ func (c *gitFetchProject) Execute(ctx context.Context,
 			continue
 		}
 
-		moduleBase := filepath.Join(module.Prefix, module.Name)
+		moduleBase := filepath.ToSlash(filepath.Join(module.Prefix, module.Name))
```

Comment:
What's the purpose of `ToSlash`? I'm curious if this is problematic on Windows somehow.